Jupyter notebook support (#2357)
psf/black: README.md

_Black_ can be installed by running `pip install black`. It requires Python 3.6.2+ to
run. If you want to format Python 2 code as well, install with

> `pip install black[python2]`. If you want format Jupyter Notebooks as well, install with

Reviewer: ```suggestion
`pip install black[python2]`. If you want to format Jupyter Notebooks, install with
```
